src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let statedCopy = { ...state };
+  const addProp = 'addProperties';
+  const removeProp = 'removeProperties';
+  const clear = 'clear';
```

Comment:
Would be better to refactor constant variables to uppercase letters like this: `addprop => ADD_PROP`